doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,229 @@
+            set initial refinement = 0
+        end
+    end
+
+Immersed Boundary Particles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We define the particle in flow using the sharp immersed boundary method. The particle is defined as a cylinder with a radius of :math:`D/2 = 0.5` and a half-length of :math:`L/2 = 1.0`. The particle is centered at :math:`(0,0,0)` 
+and the cylider's axis is oriented perpendicularly to direction of the flow. 
+
+The ``local mesh refinement`` is set to 3 to ensure we resolve the boundary layer flow around the particle with elements of size :math:`1/32 D \times 1/32 D \times 1/32 D`.
+
+.. code-block:: text
+
+    subsection particles
+        set number of particles                     = 1
+        set assemble Navier-Stokes inside particles = false
+        subsection extrapolation function
+            set stencil order = 2
+            set length ratio  = 1
+        end
+        subsection local mesh refinement
+            set initial refinement                = 3
+            set refine mesh inside radius factor  = 0.75
+            set refine mesh outside radius factor = 2
+        end
+        subsection particle info 0
+            set type            = cylinder
+            set shape arguments = 0.5;1.0
+            set integrate motion = false
+            subsection position
+                set Function expression = 0;0;0
+            end
+            subsection orientation
+                set Function expression = 1.5707963267948966;0;0
+            end
+        end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+We define the boundary conditions similarly to the :doc:`../../incompressible-flow/3d-flow-around-sphere/flow-around-sphere` example although, in our case, we won't need a boundary condition defined on the particle as the sharp immersed boundary method will take care of it.
+
+The domain makes use of an inlet velocity of :math:`1~m/s` on the left, slip boundary conditions parallel to the flow direction, and an `outlet` boundary condition, corresponding to :math:`p = 0`, at the right of the domain.
+
+.. code-block:: text
+
+    subsection boundary conditions
+        set number = 6
+        subsection bc 0
+            set id = 0
+            set type = function
+            subsection u
+                set Function expression = 1
+            end
+            subsection v
+                set Function expression = 0
+            end
+            subsection w
+                set Function expression = 0
+            end
+        end
+        subsection bc 1
+            set id = 1
+            set type = outlet
+        end
+        subsection bc 2
+            set id = 2
+            set type = slip
+        end
+        subsection bc 3
+            set id = 3
+            set type = slip
+        end
+        subsection bc 4
+            set id = 4
+            set type = slip
+        end
+        subsection bc 5
+            set id = 5
+            set type = slip
+        end
+    end
```

Comment:
Please use the prmindent tool (lethe/lethe/contrib/utilities/prmindent) to indent the entries of this subsection correctly (both in the .rst file and the .prm file)